src/oca_pre_commit_hooks/checks_odoo_module_xml.py

```diff
@@ -348,3 +348,18 @@ def check_xml_xpath(self):
                         f'{manifest_data["filename_short"]}:{xpath_node.sourceline} '
                         f"Use of translatable xpath `text()`"
                     )
+
+    @utils.only_required_for_checks("xml-oe-structure-missing-id")
+    def check_xml_oe_structure(self):
+        """* Check xml-oe-structure-missing-id
+
+        Ensure all tags with class 'oe_structure' have an ID. For more information on the rationale see:
```

Comment:
```suggestion
        Ensure all tags with class 'oe_structure' have an ID. For more information on the rationale, see:
```

Same in the other occurrence